### PR TITLE
chore: added comment on try/catch block

### PR DESCRIPTION
**Pull request by munkhuushmgl**
Added comment on on every try-with-resources statement.
Updated system.out.printf --> system.out.format new Format

### CODE REVIEW

File: vision/cloud-client/src/main/java/com/example/vision/Detect.java [hunk 1]
```diff
@@ -95,19 +95,19 @@ public static void main(String[] args) throws Exception, IOException {
...
-              + "Path to File:\n\tA path to the remote file on Cloud Storage (gs://...)\n"
-              + "Path to Destination\n\tA path to the remote destination on Cloud Storage for the"
-              + " file to be saved. (gs://BUCKET_NAME/PREFIX/)\n");
+              + "| web | web-entities | web-entities-include-geo | crop | ocr %n"
+              + "| object-localization %n"
+              + "Path:%n\tA file path (ex: ./resources/wakeupcat.jpg) or a URI for a Cloud Storage "
+              + "resource (gs://...)%n"
+              + "Path to File:%n\tA path to the remote file on Cloud Storage (gs://...)%n"
+              + "Path to Destination%n\tA path to the remote destination on Cloud Storage for the"
+              + " file to be saved. (gs://BUCKET_NAME/PREFIX/)%n");
```
Comment (lesv) [inline vision/cloud-client/src/main/java/com/example/vision/Detect.java:110]: Ok, but this is no longer the standard we use.

Comment (lesv) [inline vision/cloud-client/src/main/java/com/example/vision/Detect.java:110]: Why isn't this just a bunch of out.println(...);  ??

Comment (munkhuushmgl) [reply vision/cloud-client/src/main/java/com/example/vision/Detect.java:110]: I will just remove this argsHelper method

**Comment by munkhuushmgl** [comment]
@lesv I started moving Detect samples into individual files, will continue moving the rest of files in different PR.

**Comment by lesv** [comment]
This one is failing, so you might wish to fix that before you do much else.

**Comment by lesv** [comment]
```
------------------------------------------------------------
- testing vision/cloud-client
------------------------------------------------------------
     [java] JVM args ignored when same JVM is used.
     [java] The following classes needed for analysis were missing:
     [java]   accept
[ERROR] Dead store to result in com.example.vision.Detect.detectDocumentsGcs(String, String) [com.example.vision.Detect] At Detect.java:[line 1180] DLS_DEAD_LOCAL_STORE
[ERROR] Found reliance on default encoding in com.example.vision.Detect.detectDocumentsGcs(String, String): new String(byte[]) [com.example.vision.Detect] At Detect.java:[line 1217] DM_DEFAULT_ENCODING
[ERROR] Method com.example.vision.Detect.detectCropHints(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 911] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectDocumentText(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 996] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectLandmarks(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 88] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectLocalizedObjects(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 1251] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectLogos(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 212] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectProperties(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 378] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectSafeSearch(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 474] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectText(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 294] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectWebDetections(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 568] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectWebEntities(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 705] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectWebEntitiesIncludeGeoResults(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.Detect] At Detect.java:[line 799] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] Method com.example.vision.Detect.detectWebEntities(String) builds a list from one element using Arrays.asList rather than Collections.singletonList [com.example.vision.Detect] At Detect.java:[line 718] LUI_USE_SINGLETON_LIST
[ERROR] Method com.example.vision.Detect.detectWebEntitiesGcs(String) builds a list from one element using Arrays.asList rather than Collections.singletonList [com.example.vision.Detect] At Detect.java:[line 764] LUI_USE_SINGLETON_LIST
[ERROR] Method com.example.vision.Detect.detectWebEntitiesIncludeGeoResults(String) builds a list from one element using Arrays.asList rather than Collections.singletonList [com.example.vision.Detect] At Detect.java:[line 821] LUI_USE_SINGLETON_LIST
[ERROR] Method com.example.vision.Detect.detectWebEntitiesIncludeGeoResultsGcs(String) builds a list from one element using Arrays.asList rather than Collections.singletonList [com.example.vision.Detect] At Detect.java:[line 879] LUI_USE_SINGLETON_LIST
[ERROR] Method com.example.vision.Detect.detectDocumentsGcs(String, String) encodes String bytes without specifying the character encoding [com.example.vision.Detect] At Detect.java:[line 1217] MDM_STRING_BYTES_ENCODING
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 911] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 996] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 88] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 1251] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 212] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 378] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 474] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 294] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 568] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 705] PATH_TRAVERSAL_IN
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.Detect] At Detect.java:[line 799] PATH_TRAVERSAL_IN
[ERROR] Method com.example.vision.Detect.detectProperties(String) appears to call the same method on the same object redundantly [com.example.vision.Detect] At Detect.java:[line 407] PRMC_POSSIBLY_REDUNDANT_METHOD_CALLS
[ERROR] Method com.example.vision.Detect.detectPropertiesGcs(String) appears to call the same method on the same object redundantly [com.example.vision.Detect] At Detect.java:[line 455] PRMC_POSSIBLY_REDUNDANT_METHOD_CALLS
[ERROR] com.example.vision.Detect.detectDocumentText(String) concatenates strings using + in a loop [com.example.vision.Detect] At Detect.java:[line 1029] SBSC_USE_STRINGBUFFER_CONCATENATION
[ERROR] com.example.vision.Detect.detectDocumentTextGcs(String) concatenates strings using + in a loop [com.example.vision.Detect] At Detect.java:[line 1095] SBSC_USE_STRINGBUFFER_CONCATENATION
[ERROR] This method com.example.vision.snippets.AsyncBatchAnnotateImages.asyncBatchAnnotateImages(String, String) is declared more permissively than is used in the code base [com.example.vision.snippets.AsyncBatchAnnotateImages] At AsyncBatchAnnotateImages.java:[lines 46-87] OPM_OVERLY_PERMISSIVE_METHOD
[ERROR] This method com.example.vision.snippets.BatchAnnotateFiles.batchAnnotateFiles(String) is declared more permissively than is used in the code base [com.example.vision.snippets.BatchAnnotateFiles] At BatchAnnotateFiles.java:[lines 49-112] OPM_OVERLY_PERMISSIVE_METHOD
[ERROR] This API (java/nio/file/Paths.get(Ljava/lang/String;[Ljava/lang/String;)Ljava/nio/file/Path;) reads a file whose location might be specified by user input [com.example.vision.snippets.BatchAnnotateFiles] At BatchAnnotateFiles.java:[line 54] PATH_TRAVERSAL_IN
[ERROR] Method com.example.vision.snippets.BatchAnnotateFiles.batchAnnotateFiles(String) appears to call the same method on the same object redundantly [com.example.vision.snippets.BatchAnnotateFiles] At BatchAnnotateFiles.java:[line 94] PRMC_POSSIBLY_REDUNDANT_METHOD_CALLS
[ERROR] This method com.example.vision.snippets.BatchAnnotateFilesGcs.batchAnnotateFilesGcs(String) is declared more permissively than is used in the code base [com.example.vision.snippets.BatchAnnotateFilesGcs] At BatchAnnotateFilesGcs.java:[lines 46-107] OPM_OVERLY_PERMISSIVE_METHOD
[ERROR] Method com.example.vision.snippets.BatchAnnotateFilesGcs.batchAnnotateFilesGcs(String) appears to call the same method on the same object redundantly [com.example.vision.snippets.BatchAnnotateFilesGcs] At BatchAnnotateFilesGcs.java:[line 89] PRMC_POSSIBLY_REDUNDANT_METHOD_CALLS
[ERROR] Method com.example.vision.snippets.DetectFaces.detectFaces(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.snippets.DetectFaces] At DetectFaces.java:[line 31] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] This method com.example.vision.snippets.DetectFaces.detectFaces(String) is declared more permissively than is used in the code base [com.example.vision.snippets.DetectFaces] At DetectFaces.java:[lines 29-63] OPM_OVERLY_PERMISSIVE_METHOD
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.snippets.DetectFaces] At DetectFaces.java:[line 31] PATH_TRAVERSAL_IN
[ERROR] This method com.example.vision.snippets.DetectFacesGcs.detectFacesGcs(String) is declared more permissively than is used in the code base [com.example.vision.snippets.DetectFacesGcs] At DetectFacesGcs.java:[lines 28-62] OPM_OVERLY_PERMISSIVE_METHOD
[ERROR] Method com.example.vision.snippets.DetectLabels.detectLabels(String) uses a FileInputStream or FileOutputStream constructor [com.example.vision.snippets.DetectLabels] At DetectLabels.java:[line 31] IOI_USE_OF_FILE_STREAM_CONSTRUCTORS
[ERROR] This method com.example.vision.snippets.DetectLabels.detectLabels(String) is declared more permissively than is used in the code base [com.example.vision.snippets.DetectLabels] At DetectLabels.java:[lines 29-58] OPM_OVERLY_PERMISSIVE_METHOD
[ERROR] This API (java/io/FileInputStream.<init>(Ljava/lang/String;)V) reads a file whose location might be specified by user input [com.example.vision.snippets.DetectLabels] At DetectLabels.java:[line 31] PATH_TRAVERSAL_IN
[ERROR] This method com.example.vision.snippets.DetectLabelsGcs.detectLabelsGcs(String) is declared more permissively than is used in the code base [com.example.vision.snippets.DetectLabelsGcs] At DetectLabelsGcs.java:[lines 29-57] OPM_OVERLY_PERMISSIVE_METHOD
[ERROR] src/main/java/com/example/vision/snippets/DetectLabels.java:[1] (header) RegexpHeader: Line does not match expected header line of '^\W+\*?\W*Copyright 20\d\d Google (Inc\.|LLC)$'.
[ERROR] src/main/java/com/example/vision/snippets/DetectLabels.java:[14] (imports) CustomImportOrder: Extra separation in import group before 'java.io.FileInputStream'
[ERROR] src/main/java/com/example/vision/snippets/DetectLabels.java:[54] (sizes) LineLength: Line is longer than 100 characters (found 103).
[ERROR] src/main/java/com/example/vision/snippets/DetectFacesGcs.java:[1] (header) RegexpHeader: Line does not match expected header line of '^\W+\*?\W*Copyright 20\d\d Google (Inc\.|LLC)$'.
[ERROR] src/main/java/com/example/vision/snippets/DetectFacesGcs.java:[14] (imports) CustomImportOrder: Extra separation in import group before 'java.io.IOException'
[ERROR] src/main/java/com/example/vision/snippets/DetectFaces.java:[1] (header) RegexpHeader: Line does not match expected header line of '^\W+\*?\W*Copyright 20\d\d Google (Inc\.|LLC)$'.
[ERROR] src/main/java/com/example/vision/snippets/DetectFaces.java:[14] (imports) CustomImportOrder: Extra separation in import group before 'java.io.FileInputStream'
[ERROR] src/main/java/com/example/vision/snippets/DetectLabelsGcs.java:[1] (header) RegexpHeader: Line does not match expected header line of '^\W+\*?\W*Copyright 20\d\d Google (Inc\.|LLC)$'.
[ERROR] src/main/java/com/example/vision/snippets/DetectLabelsGcs.java:[14] (imports) CustomImportOrder: Extra separation in import group before 'java.io.IOException'
[ERROR] src/main/java/com/example/vision/snippets/DetectLabelsGcs.java:[53] (sizes) LineLength: Line is longer than 100 characters (found 103).
[ERROR] src/main/java/com/example/vision/Detect.java:[1034] (sizes) LineLength: Line is longer than 100 characters (found 104).
[ERROR] src/main/java/com/example/vision/Detect.java:[1100] (sizes) LineLength: Line is longer than 100 characters (found 104).
[ERROR] src/test/java/com/example/vision/DetectFacesTest.java:[26] (imports) CustomImportOrder: Extra separation in import group before 'java.io.ByteArrayOutputStream'
[ERROR] src/test/java/com/example/vision/DetectFacesTest.java:[26] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.ByteArrayOutputStream' import. Should be before 'org.junit.runners.JUnit4'.
[ERROR] src/test/java/com/example/vision/DetectFacesTest.java:[27] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.PrintStream' import. Should be before 'org.junit.runners.JUnit4'.
[ERROR] src/test/java/com/example/vision/DetectFacesTest.java:[29] (imports) CustomImportOrder: Import statement for 'com.google.common.truth.Truth.assertThat' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.
[ERROR] src/test/java/com/example/vision/DetectLabelsTest.java:[26] (imports) CustomImportOrder: Extra separation in import group before 'java.io.ByteArrayOutputStream'
[ERROR] src/test/java/com/example/vision/DetectLabelsTest.java:[26] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.ByteArrayOutputStream' import. Should be before 'org.junit.runners.JUnit4'.
[ERROR] src/test/java/com/example/vision/DetectLabelsTest.java:[27] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.PrintStream' import. Should be before 'org.junit.runners.JUnit4'.
[ERROR] src/test/java/com/example/vision/DetectLabelsTest.java:[29] (imports) CustomImportOrder: Import statement for 'com.google.common.truth.Truth.assertThat' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.
[ERROR] src/test/java/com/example/vision/BatchAnnotateFilesTest.java:[25] (imports) CustomImportOrder: Extra separation in import group before 'com.example.vision.snippets.BatchAnnotateFiles'
[ERROR] src/test/java/com/example/vision/BatchAnnotateFilesTest.java:[25] (imports) CustomImportOrder: Wrong lexicographical order for 'com.example.vision.snippets.BatchAnnotateFiles' import. Should be before 'java.io.PrintStream'.
[ERROR] src/test/java/com/example/vision/DetectLabelsGcsTest.java:[26] (imports) CustomImportOrder: Extra separation in import group before 'java.io.ByteArrayOutputStream'
[ERROR] src/test/java/com/example/vision/DetectLabelsGcsTest.java:[26] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.ByteArrayOutputStream' import. Should be before 'org.junit.runners.JUnit4'.
[ERROR] src/test/java/com/example/vision/DetectLabelsGcsTest.java:[27] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.PrintStream' import. Should be before 'org.junit.runners.JUnit4'.
[ERROR] src/test/java/com/example/vision/DetectLabelsGcsTest.java:[29] (imports) CustomImportOrder: Import statement for 'com.google.common.truth.Truth.assertThat' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.
[ERROR] src/test/java/com/example/vision/DetectLabelsGcsTest.java:[55,36] (whitespace) ParenPad: '(' is followed by whitespace.
[ERROR] src/test/java/com/example/vision/BatchAnnotateFilesGcsTest.java:[25] (imports) CustomImportOrder: Extra separation in import group before 'com.example.vision.snippets.BatchAnnotateFilesGcs'
[ERROR] src/test/java/com/example/vision/BatchAnnotateFilesGcsTest.java:[25] (imports) CustomImportOrder: Wrong lexicographical order for 'com.example.vision.snippets.BatchAnnotateFilesGcs' import. Should be before 'java.io.PrintStream'.
[ERROR] src/test/java/com/example/vision/SetEndpointIT.java:[25] (imports) CustomImportOrder: Extra separation in import group before 'com.example.vision.snippets.SetEndpoint'
[ERROR] src/test/java/com/example/vision/SetEndpointIT.java:[25] (imports) CustomImportOrder: Wrong lexicographical order for 'com.example.vision.snippets.SetEndpoint' import. Should be before 'java.io.PrintStream'.
[ERROR] src/test/java/com/example/vision/DetectIT.java:[224,46] (whitespace) ParenPad: '(' is followed by whitespace.
[ERROR] src/test/java/com/example/vision/DetectIT.java:[235] (sizes) LineLength: Line is longer than 100 characters (found 103).
[ERROR] src/test/java/com/example/vision/DetectIT.java:[315,34] (whitespace) ParenPad: '(' is followed by whitespace.
[ERROR] src/test/java/com/example/vision/DetectIT.java:[325] (sizes) LineLength: Line is longer than 100 characters (found 103).
[ERROR] src/test/java/com/example/vision/DetectFacesGcsTest.java:[26] (imports) CustomImportOrder: Extra separation in import group before 'java.io.ByteArrayOutputStream'
[ERROR] src/test/java/com/example/vision/DetectFacesGcsTest.java:[26] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.ByteArrayOutputStream' import. Should be before 'org.junit.runners.JUnit4'.
[ERROR] src/test/java/com/example/vision/DetectFacesGcsTest.java:[27] (imports) CustomImportOrder: Wrong lexicographical order for 'java.io.PrintStream' import. Should be before 'org.junit.runners.JUnit4'.
[ERROR] src/test/java/com/example/vision/DetectFacesGcsTest.java:[29] (imports) CustomImportOrder: Import statement for 'com.google.common.truth.Truth.assertThat' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.1:check (default) on project vision-google-cloud-samples: You have 39 Checkstyle violations. -> [Help 1]
[ERROR]
```

**Pull request merged**